source/analysis/AM3DAdditionAB.cpp

```diff
@@ -253,5 +257,6 @@ void AM3DAdditionAB::computeCachedValues() const
             cachedData_[j] += data.at(j);
     }
 
+    cachedDataRange_ = AMUtility::rangeFinder(cachedData_);
```

Comment:
Unfortunately not :(  A default that was implemented by default would be great, but we've mostly just been getting along by trying to assume that -1 is "not too bad".  Far from ideal but mostly a legacy design decision/hack.